plugins/base/src/main/kotlin/resolvers/external/javadoc/JavadocExternalLocationProviderFactory.kt

```diff
@@ -11,15 +14,21 @@ class JavadocExternalLocationProviderFactory(val context: DokkaContext) :
     ExternalLocationProviderFactory by ExternalLocationProviderFactoryWithCache(
         object : ExternalLocationProviderFactory {
             override fun getExternalLocationProvider(doc: ExternalDocumentation): ExternalLocationProvider? =
-                when (doc.packageList.linkFormat) {
-                    RecognizedLinkFormat.Javadoc1 ->
-                        JavadocExternalLocationProvider(doc, "()", ", ", context) // Covers JDK 1 - 7
-                    RecognizedLinkFormat.Javadoc8 ->
-                        JavadocExternalLocationProvider(doc, "--", "-", context) // Covers JDK 8 - 9
-                    RecognizedLinkFormat.Javadoc10,
-                    RecognizedLinkFormat.DokkaJavadoc ->
-                        JavadocExternalLocationProvider(doc, "()", ",", context) // Covers JDK 10
-                    else -> null
+                when (doc.packageList.url) {
+                    DokkaConfiguration.ExternalDocumentationLink.Companion.androidX().packageListUrl,
```

Comment:
NIT: Don't write out the Companion here 🙄

Reply:
Oops, haven't noticed it when IntelliJ imported it by itself. Fixed 👍